src/System.Security.Cryptography.Cng/src/Internal/Windows/Crypt32/Interop.Crypt32.HashIdAlg.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.NativeCrypto;
+using System;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+
+internal partial class Interop
+{
+    internal static partial class Crypt32
+    {
+        private const string Capi2Dll = "Crypt32.dll";
+
+        /// <summary>
+        /// Version used for a buffer containing a scalar integer (not an IntPtr)
+        /// </summary>
+        [DllImport(Capi2Dll, CharSet = CharSet.Unicode, EntryPoint = "CryptFindOIDInfo")]
+        private static extern IntPtr CryptFindOIDInfoByInt(CryptOidInfoKeyType dwKeyType, ref int pvKey, OidGroup group);
```

Comment:
Since your public method doesn't start with "Crypt", do you need the "Int" suffix and the EntryPoint value?